index.html

```diff
@@ -33,13 +63,45 @@ <h1 class="logo-heading">Fun Bus</h1>
         </div>
       </div>
     </div>
-  </div>
+  </div>-->
   <div class="container">
-    <div class="row">
+    <div class="column">
         <div class="col-sm-12">
             <img src="img/fun-bus.jpg" class="img-fluid" alt="Responsive image">
         </div>
     </div>
+    <!--navbar-->
+    <div class="container-fluid sticky-top"></div>
+    <div class= "container">
+<div class="d-flex justify-content-end">
```

Comment:
Be careful with your indentation here, taking care to nest blocks of code correctly. This will become more and more important as your code becomes longer and more complex.